soc/silabs/silabs_siwx91x/siwg917/siwx91x_nwp.c

```diff
@@ -503,6 +523,7 @@ BUILD_ASSERT(CONFIG_SIWX91X_NWP_INIT_PRIORITY < CONFIG_KERNEL_INIT_PRIORITY_DEFA
 		.enable_xtal_correction = DT_INST_PROP(inst, enable_xtal_correction),              \
 		.qspi_80mhz_clk = DT_INST_PROP(inst, qspi_80mhz_clk),                              \
 		.antenna_selection = DT_INST_ENUM_IDX(inst, antenna_selection),                    \
+		.clock_frequency = DT_INST_PROP_OR(inst, clock_frequency, 0)                      \
```

Comment:
If you define the `default` attribute in the binding, you don't have to repeat the default value here.